sklearn/decomposition/incremental_pca.py

```diff
@@ -204,42 +209,44 @@ def partial_fit(self, X, y=None):
         else:
             self.n_components_ = self.n_components
 
-        if (self.components_ is not None) and (self.components_.shape[0]
-                                               != self.n_components_):
+        if (self.components_ is not None) and (self.components_.shape[0] !=
+                                               self.n_components_):
             raise ValueError("Number of input features has changed from %i "
                              "to %i between calls to partial_fit! Try "
-                             "setting n_components to a fixed value." % (
-                                 self.components_.shape[0], self.n_components_))
+                             "setting n_components to a fixed value." %
+                             (self.components_.shape[0], self.n_components_))
 
+        # This is the first partial_fit
```

Comment:
I don't remember what the consensus was wrt calling ``fit`` and then ``partial_fit``. Should we forget the model? I think there are some arguments about that in #3907.